### DISCUSSION TITLE
Scrollabar on bottom when loading new records

### DISCUSSION THREAD

**Discussion by mscarda** [TomSchimansky/CustomTkinter#1648] [Q&A] [answered]
Hi everybody, i'm using CTKScrollableFrame to create a log interface.
When it populate a new record, instead to scroll down to view it, I would like to have it already loaded on bottom.
Any thoughts?
Thanks in advance

**Comment by Akascape** [comment, marked as answer]
@mscarda 
Use this command:
```python
scroll_frame._parent_canvas.yview_moveto(1.0)
```
(here, **scroll_frame** is your CTkScrollableFrame widget).

**Reply by mscarda** [reply]
Thanks a lot, works like a charm!
